gen_gebieden.py

```diff
@@ -0,0 +1,347 @@
+from generate import block, header, p, q
+
+
+# name, group, classname, table, color, minscale, midscale, maxscale, abstract
+#
+# Scales are for the labels only.
+layers = [
+    (
+        "buurt",
+        "bestuurlijke_gebieden",
+        "Buurt",
+        "buurten",
+        "210 150 50",
+        100,
+        6001,
+        8001,
+        "Buurten van de gemeente Amsterdam",
+    ),
+    # The *_simple layers exist for backward compatibility only.
+    (
+        "buurt_simple",
+        "bestuurlijke_gebieden",
+        "Buurt",
+        "buurten",
+        "210 150 50",
+        100,
+        6001,
+        8001,
+        "Buurten van de gemeente Amsterdam",
+    ),
+    # De historic name "buurtcombinatie" is used for wijken because
+    # data.amsterdam.nl still depends on it.
+    (
+        "buurtcombinatie",
+        "bestuurlijke_gebieden",
+        "Wijk",
+        "wijken",
+        "210 100 50",
+        5000,
+        10001,
+        15001,
+        "Wijken van de gemeente Amsterdam",
```

Comment:
Is it intentional to use Wijken and not buurtcombinaties?

Reply:
Yes. See comment above: wijken is the new name, buurtcombinaties is there for backwards compatibility. This was already the case in the old mapfile.